src/sliceDataStorage.h

```diff
@@ -67,8 +69,18 @@ class SliceLayerPart
     /*!
      * The areas which need to be filled with sparse (0-99%) infill for different thicknesses.
      * The infill_area is an array to support thicker layers of sparse infill and areas of different infill density.
+     *
+     * Take an example of infill_area[x][n], the meanings of the indexes are as follows:
+     *   x  -  The sparsity of the infill area (0 - 99 in percentage). So, the areas in infill_area[x] are the most dense ones.
+     *   n  -  The thickness (in number of layers) of the infill area. See example below:
+     *          / ------ -- /         <- layer 3
+     *        /-- ------ /            <- layer 2
+     *          1   2     3
+     * LEGEND: "-" means infill areas.
+     * Numbers 1, 2 and 3 identifies 3 different infill areas. Infill areas 1 and 3 have a tickness of 1 layer, while area 2 has a thickness of 2.
```

Comment:
You read my comment wrong.
I was talking about the two 2's on the **left** of *my exmaple*.